typescript/web/src/components/website/Movie/movie.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Heading, Flex, Center, AspectRatio } from "@chakra-ui/react";
+import * as React from "react";
+
+export const Movie = () => {
```

Comment:
Here is what our code-quality tool warns for this file:
![image](https://user-images.githubusercontent.com/2091902/155103203-dc853457-9260-4634-b2c6-85e50e7bed2c.png)

Can we merge the code with `components/website/Video/movie.tsx` and split it a litle bit (see my suggestions below)?

Reply:
It is good except I do not succeed in centering the video...